package.json

```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "bluebird": "^3.5.1",
+    "csv": "^3.0.2",
```

Comment:
Is this extra dependency truly necessary?
Can't we split a csv simply by using something like `csv.split('\n').map((s) => s.split(',');`?

Reply:
I'll try to do the "mini parsing" (your code is very simple... its missing the column names for the objects, etc) but no problem!